### PR TITLE
(feat) 041: react presentation

### PR DESCRIPTION
**Pull request by Dzmitry-Klokau**

### CODE REVIEW

File: react/presentation.html [hunk 1]
```diff
@@ -2,43 +2,4710 @@
...
       <!-- Any section element inside of this container is displayed as a slide -->
       <div class="slides">
-        <section class="center">
-          <h2 class="custom-presentation-name"> Presentation​ Name </h3>
+        <!-- 1-ST SLIDE -->
+        <section class="presentation-name-slide">
+          <h4>React</h4>
+
+          <aside class="notes">
+            <strong>ATTENSION SUPER IMPORTANT LECTION</strong>
```
Comment (unfalse) [inline react/presentation.html:49]: Misspell - ATTENTION

**Comment by unfalse** [comment]
Videos look tiny on my FullHD display

Screen Shot 2022-11-18 at 10 55 57

File: react/presentation.html [hunk 2]
```diff
@@ -2,43 +2,4710 @@
...
+            <div>
+              <img
+                src="assets/images/custom_hook_example.png"
+                alt="custom_hook_example"
+              />
+              <ol>
+                <li><strong>use-custom-hook-component.js</strong> file</li>
+              </ol>
+            </div>
+            <aside class="notes">
```
Comment (unfalse) [inline react/presentation.html:3277]: how to open the text from aside tag? I can't see it on running presentation

File: react/presentation.html [hunk 3]
```diff
@@ -2,43 +2,4710 @@
...
+                }              
+					    </code>
+            </pre>
+            <aside class="notes">
+              And the last thing we need to do is add our new component to
+              App.js.
+            </aside>
+          </section>
+          <section class="video-slide">
+            <h4 class="slide-name">useEffect example</h4>
```
Comment (unfalse) [inline react/presentation.html:2563]: it should be useCallback

File: react/presentation.html (hunk 2 above)
Comment (Dzmitry-Klokau) [reply react/presentation.html:3277]: you should add ```?showNotes``` to the url

**Review by Karina-Trosko — APPROVED** [review]

**Review by annakosakova — APPROVED** [review]

**Pull request merged**